tests/test_bundle.py

```diff
@@ -47,7 +48,12 @@
 ISTIO_GATEWAY_APP_NAME = "istio-ingressgateway"
 ISTIO_GATEWAY_TRUST = True
 
+CLOUDEVENTS_MANIFEST = lightkube.codecs.load_all_yaml(
+    Path("./examples/cloudevents-player.yaml").read_text()
+)
+
 
+@pytest.mark.skip_if_deployed
```

Comment:
Is this a leftover while you were debugging? (:

Reply:
yes :) removing it